examples/complicated_cpu_bound_benchmark.cf

```diff
@@ -0,0 +1,63 @@
+body common control
+{
+      bundlesequence => { "benchmark" };
+}
+
+bundle agent benchmark
+{
+  vars:
+    FILES::
+      "testdir" string => "/tmp/testdir";
```

Comment:
@larsewi as discussed in the ticket, I think it's best to leave this out for another test that is not the _CPU bound_ benchmark.